example/ConnectionInfoFetch.tsx

```diff
@@ -0,0 +1,45 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @format
+ */
+
+import * as React from 'react';
+import {Text, View, TouchableOpacity} from 'react-native';
+import NetInfo, {NetInfoSubscription} from '../src';
+
+interface State {
+  connectionInfo: string;
+}
+
+export default class ConnectionInfoCurrent extends React.Component<{}, State> {
+  _subscription: NetInfoSubscription | null = null;
```

Comment:
`_subscription` is not used in this component

Reply:
Good catch!